snap/hooks/configure

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# this is a little hackish, but reads better to use string concatenation 
+# to build the list of services than have a long string 
+ALL_SERVICES=""
+
+# base services
+ALL_SERVICES="$ALL_SERVICES consul"
+ALL_SERVICES="$ALL_SERVICES mongod"
+ALL_SERVICES="$ALL_SERVICES mongo-worker"
+
+# core services
+ALL_SERVICES="$ALL_SERVICES core-data"
+ALL_SERVICES="$ALL_SERVICES core-metadata"
+ALL_SERVICES="$ALL_SERVICES core-config-seed"
+ALL_SERVICES="$ALL_SERVICES core-command"
+
+# support services
+ALL_SERVICES="$ALL_SERVICES support-notifications"
+ALL_SERVICES="$ALL_SERVICES support-logging"
+ALL_SERVICES="$ALL_SERVICES support-scheduler"
+ALL_SERVICES="$ALL_SERVICES support-rulesengine"
```

Comment:
We just removed support-rulesengine in the previous snap merged, so this should be dropped here as well.

Reply:
Good catch